Block/Bordered/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../css/settings';
+@import '../../css/mixins/index';
+
+.block--bordered {
+  background: map-get($colors, white);
+  border: ($grid * .2) solid $border-color;
+  border-radius: $border-radius;
+  box-sizing: border-box;
```

Comment:
It’s still adding up to one more than the grid unfortunately. Let’s set one pixel less in the top padding:

`paddingTop: ($grid * 2.8)`

I also noticed that the margins on top and on bottom, when added to the default text paddings, make the vertical spacings look bigger than the horizontal ones. What if we make the top and bottom paddings one grid less?

```css
paddingBottom: ($grid * 2);
paddingLeft: ($grid * 3);
paddingRight: ($grid * 3);
paddingTop: ($grid * 1.8);
```